keyboards/ymdk_bface/info.json

```diff
@@ -0,0 +1,12 @@
+{
+  "keyboard_name": "YMDK Bface",
+  "url": "",
+  "maintainer": "qmk",
+  "width": 15,
+  "height": 5,
+  "layouts": {
+      "LAYOUT_60_ansi": {
```

Comment:
`LAYOUT_all` and `LAYOUT_60_ansi_split_rshift` should also have objects in this file.

Reply:
I added the other layouts. For LAYOUT_all, not all the key positions can fit together at once realistically, but I still put all the positions in. Hope that is OK.

Reply:
> For LAYOUT_all, not all the key positions can fit together at once realistically, but I still put all the positions in. Hope that is OK.

That's actually exactly what we want to be done in that situation. :)

https://docs.qmk.fm/#/hardware_keyboard_guidelines?id=ltkeyboard_namehgt

And you can ignore the part about the "additional keymaps named `default_<layout>`". They're nice to have, but we don't actually strictly enforce that.